logging/log.go

```diff
@@ -0,0 +1,52 @@
+package logging
+
+import (
+	"os"
+
+	"github.com/rs/zerolog"
+)
+
+// TODO: Should this be a pointer?
+// https://github.com/rs/zerolog/issues/1
+var Logger zerolog.Logger
```

Comment:
I don't think this should be a pointer. zerolog copies loggers when creating sub-loggers, rather than using references.